### DISCUSSION TITLE
OpenMCT on CLoud

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/openmct#5078] [Community Q&A]
Hello

Any version of OpenMCT already deployed in the cloud platforms (AWS, Azure, etc) to try demo or ready to use

**Comment by milesbjackson** [comment]
I am working on one. In the stage of getting ports set up and secure.

Having to change some things from local host to the right host ips etc.

**Comment by jvigliotta** [comment]
@satzguyz We used to have a demo up, but it became quite old, so we've taken that down, but getting a new one up is on our list of things to do. I will try to update here once we have something new up.
